internal/pkg/cli/app_deploy.go

```diff
@@ -426,10 +430,10 @@ func (opts appDeployOpts) applyAppDeployTemplate(template, stackName, changeSetN
 func (opts appDeployOpts) getAppDockerfilePath() (string, error) {
 	manifestFileNames, err := opts.workspaceService.ListManifestFiles()
 	if err != nil {
-		return "", err
+		return "", fmt.Errorf("list local manifest files: %w", err)
 	}
 	if len(manifestFileNames) == 0 {
-		return "", errors.New("no manifest files found")
+		return "", errNoLocalManifestsFound
```

Comment:
Just curious is there any special reason why we want this error to be defined outside as a variable?

Reply:
It reduces duplication in the test code when I want to assert that this specific error is returned.